index.html

```diff
@@ -61,9 +60,152 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+                        </div>
+                        <p class="">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <div class="lower_part">
+                            <ul class="project-tech-stack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectfour.png">
+                            <h3 class="ptitle">Multi-Post Stories</h3>
+                        </div>
+                        <div class="p-details">
+                            <ul>
+                                <li>CANOPY</li>
+                                
+                                <li>Back End Dev</li>
+                               
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="lower_part">
+                        <ul class="project-tech-stack">
+                            <li><a>HTML</a></li>
+                            <li><a>CSS</a></li>
+                            <li><a>Javascript</a></li>
+                        </ul>
+                        <a class="button">See project</a>
+                        </div>
+                    </section>
+            </section>
+            
+            
+            <section class="about-me">
+                <h2 class="titletext">About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="letconnect"> LET’S CONNECT</p>
+                           
+                       
+
+                    
+                <div class="socialmedia">
+                    <ul class="media">                           
+                        
+                        <li><a class="media"><img src="img/media-3.png"></a></li>
+                        <li><a class="media"><img src="img/media-4.png"></a></li>
+                        <li><a class="media"><img src="img/media-1.png"></a></li>
+                        <li><a class="media"><img src="img/media-5.png"></a></li>
+                        <li><a class="media"><img src="img/media-2.png"></a></li>
+                    </ul>
+
+                </div>
+
+                <a class="button">Get my resume</a>
```

Comment:
Great work on adding the button and implementing the active and hover states! :+1: However, there are a few adjustments needed to align it with the design:

- [ ] Ensure that proper spacing is added around the button. The spacing should be consistent with the design specifications, both horizontally and vertically. This will create a balanced and visually appealing button.

- [ ] Pay attention to the alignment of the button text. Make sure it is properly centered within the button and has appropriate padding. Consistent alignment will contribute to a polished and professional appearance.

- [ ] Review the font styles of the button, including font size, font weight, and font family. Ensure that they match the design guidelines for a cohesive and consistent look.

By updating the spacing, alignment, and font styles of the button, you will bring it closer to the design and enhance its visual appeal. Keep up the great work and strive for design accuracy!